embedded-oracle-xe/src/main/java/com/playtika/test/oracle/EmbeddedOracleBootstrapConfiguration.java

```diff
@@ -58,7 +58,8 @@ public OracleContainer oracle(ConfigurableEnvironment environment,
                         .withUsername(properties.getUser())
                         .withPassword(properties.getPassword())
                         .withLogConsumer(containerLogsConsumer(log))
-                        .withStartupTimeout(properties.getTimeoutDuration());
+                        .withStartupTimeout(properties.getTimeoutDuration())
+                        .withInitScript(properties.initScriptPath);
```

Comment:
is it okay to have ```null``` as bypassed parameter?

Reply:
If this value is null, the database script will not be executed

Reply:
not sure it worth adding such complexity, if ```MySQLContainer``` works fine with null, im okay with that.
Sure we can add test for this, but for such kind of test we need to boot additional container...